.ci/scripts/dist_s1/compare_dist_s1_products.sh

```diff
@@ -0,0 +1,170 @@
+#!/bin/bash
+
+# Script used to orchestrate the pairwise comparison of output and expected
+# DIST-S1 products. Each individual pair of products is compared using the
+# dist_s1_compare.py script.
+
+set -e
+umask 002
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+. "$SCRIPT_DIR"/../util/test_int_util.sh
+
+OUTPUT_DIR="/home/ops/output_dir"
+EXPECTED_DIR="/home/ops/expected_output_dir"
+PGE_NAME="dist_s1"
+
+# Validate that OUTPUT_DIR and EXPECTED_DIR exist within the container
+if [ ! -d "$OUTPUT_DIR" ]; then
+    echo "Error: Output directory '$OUTPUT_DIR' does not exist." >&2
+    exit 1
+fi
+
+if [ ! -d "$EXPECTED_DIR" ]; then
+    echo "Error: Expected directory '$EXPECTED_DIR' does not exist." >&2
+    exit 1
+fi
+
+initialize_html_results_file "$OUTPUT_DIR" "$PGE_NAME"
+
+# overall_status values and their meaning
+# 0 - pass
+# 1 - failure to execute some part of this script
+# 2 - product validation failure
+overall_status=0
+
+
+# TODO: Should the PGE flatten the output?
+
+# TODO: Just realized the below loop doesn't really work with the current compare script (but should be kept since
+#       I think the compare script should check by band and not whole product)
```

Comment:
I think its alright to let the SAS validation routine work on a directory (or granule) at time for us if that's what makes the most sense. In the meantime, can we remove the commented out block before this branch is merged?

Reply:
I cut out this section.

I did make a feature request for ADT to support comparisons by band (might make it easier for them/us to diagnose issues if something goes wrong) + more descriptive reasons for what is unequal.

opera-adt/dist-s1#51